deploy/charts/cert-manager/values.schema.json

```diff
@@ -0,0 +1,886 @@
+{
```

Comment:
Similarly, it'd be nice to add a header comment exaplaining what autogenerates this file and what consumes it.

Reply:
In json its not that easy to add a comment; you would have to add an attribute in the json struct. https://stackoverflow.com/a/244858